cisco_sdwan/assets/monitors/device_unreachable.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 2,
+  "created_at": "2024-04-05",
+  "last_updated_at": "2024-04-05",
+  "title": "Cisco SD-WAN Device is unreachable",
+  "tags": [
+    "integration:cisco-sdwan"
+  ],
+  "description": "This monitor checks the reachability of Cisco SD-WAN devices within a specified namespace. It triggers an alert if a device becomes unreachable, indicating potential network connectivity issues.",
+  "definition": {
+	"name": "[Cisco SD-WAN] Device unreachable alert on {{hostname.name}} in namespace {{device_namespace.name}}",
+	"type": "query alert",
+	"query": "avg(last_5m):max:cisco_sdwan.device.reachable{*} by {hostname,system_ip,device_namespace} < 0.8",
```

Comment:
Just double checking that this metric is fully defined by `hostname, system_ip, device_namespace`, there are no other sub-tags.
And hence the `max:` aggregation here doesn't do anything, am I correct?

Reply:
Exactly, it's actually fully defined by `(device_nampespace, system_ip)`